analyzer/runtime/runtime.go

```diff
@@ -88,14 +93,15 @@ func NewRuntimeAnalyzer(nodeCfg config.NodeConfig, cfg *config.BlockBasedAnalyze
 		Source: client,
 	}
 
-	qf := analyzer.NewQueryFactory(strcase.ToSnake(nodeCfg.ChainID), emerald.String())
+	qf := analyzer.NewQueryFactory(strcase.ToSnake(nodeCfg.ChainID), runtime.String())
 
 	return &Main{
+		runtime: runtime,
 		cfg:     ac,
 		qf:      qf,
 		target:  target,
-		logger:  logger.With("analyzer", EmeraldDamaskAnalyzerName),
-		metrics: metrics.NewDefaultDatabaseMetrics(EmeraldDamaskAnalyzerName),
+		logger:  logger.With("analyzer", runtime.String()+"_"+MajorCompat),
+		metrics: metrics.NewDefaultDatabaseMetrics(runtime.String() + "_" + MajorCompat),
```

Comment:
Let's get rid of the `MajorCompat` const altogether, here and elsewhere.
The analyzer is and will be independent of the underlying storage.
We don't cosume these prometheus metrics yet so it's easy to change the name now, but we'll soon base some alerts on them.

Reply:
:raised_hands: k

Reply:
removed. y'all metrics-lookers-at are hereby warned